sycl/test-e2e/Basic/built-ins/const_vec_math.cpp

```diff
@@ -2,8 +2,8 @@
 
 // RUN: %{build} %{mathflags} -o %t.out
 // RUN: %{run} %t.out
-// RUN: %if preview-breaking-changes-supported %{ %{build} -fpreview-breaking-changes %{mathflags} -o %t2.out %}
-// RUN: %if preview-breaking-changes-supported %{ %{run} %t2.out %}
+// RUN: %{build} %{mathflags} -o %t2.out
+// RUN: %{run} %t2.out
```

Comment:
Does it make sense to run the test twice?

Reply:
Added missed `-D__SYCL_USE_LIBSYCL8_VEC_IMPL=1` option (we still have old vec implementation in place, so we run the test for both old and new)